recipes/kafka-rest.rb

```diff
@@ -43,7 +44,7 @@
 end
 
 service 'kafka-rest' do
-  action [:enable, :start]
+  action %i[enable start]
```

Comment:
Have you ever seen a good reason why this notation tends to be preferred?  I prefer the explicit array myself but have seen style guides say to prefer the % notation without really explaining why :(